.github/workflows/reusable-e2e.yml

```diff
@@ -154,10 +154,6 @@ jobs:
         if: ${{ matrix.package == 'react' }}
         run: yarn react-core build
 
-      - name: Build react-core-notifications package
```

Comment:
Can you explain why this is no longer needed?

Reply:
We do not need to build `react-core-notifications` for `ui-react` anymore as we moved inappmessaging code out and do not have any code depending on `react-core-notifications` now.

Earlier, inappmessaging code in `ui-react` depended on `react-core-notifications`.

Reply:
Added this back as we need to build `react-core-notifications` for example app to use.